streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -40,25 +40,23 @@ public KStreamBuilder() {
     }
 
     /**
-     * Creates a KStream instance for the specified topic.
+     * Creates a {@link KStream} instance from the specified topics.
      * The default deserializers specified in the config are used.
      *
-     * @param topics          the topic names, if empty default to all the topics in the config
-     * @return KStream
+     * @param topics    the topic names, if empty default to all the topics in the config
      */
     public <K, V> KStream<K, V> stream(String... topics) {
         return stream(null, null, topics);
     }
 
     /**
-     * Creates a KStream instance for the specified topic.
+     * Creates a {@link KStream} instance for the specified topics.
      *
-     * @param keySerde key serde used to read this source KStream,
-     *                 if not specified the default serde defined in the configs will be used
-     * @param valSerde value serde used to read this source KStream,
-     *                 if not specified the default serde defined in the configs will be used
-     * @param topics   the topic names, if empty default to all the topics in the config
-     * @return KStream
+     * @param keySerde  key serde used to read this source {@link KStream},
+     *                  if not specified the default serde defined in the configs will be used
+     * @param valSerde  value serde used to read this source {@link KStream},
+     *                  if not specified the default serde defined in the configs will be used
+     * @param topics    the topic names, if empty default to all the topics in the config
```

Comment:
"if empty default to all the topics in the config" -- wait, which configuration option is that?